MOBILE_API.xml

```diff
@@ -2608,6 +2608,10 @@
         <element name="UnsubscribeWayPointsID" value="47" hexvalue="2F" since="4.1" />
         <element name="GetSystemCapabilityID" value="48" hexvalue="30" since="4.5" />
         <element name="SendHapticDataID" value="49" hexvalue="31" since="4.5" />
+        <element name="PublishAppServiceID" value="52" hexvalue="34" since="5.1" />
+        <element name="GetAppServiceDataID" value="53" hexvalue="35" since="5.1" />
+        <element name="GetFileID" value="54" hexvalue="36" since="5.1" />
+        <element name="PerformAppServiceInteractiontFileID" value="55" hexvalue="37" since="5.1" />
```

Comment:
Typo, shouldn't have File attached to name should be PerformAppServiceInteractiontID